src/components/NumberInput/NumberInput-test.js

```diff
@@ -187,8 +192,8 @@ describe('NumberInput', () => {
       );
 
       const input = wrapper.find('input');
-      const upArrow = wrapper.find('.up-icon').last();
-      const downArrow = wrapper.find('.down-icon').last();
+      const upArrow = wrapper.find('Icon.up-icon').closest('button');
```

Comment:
this would use parent as well but there is a bug in enzyme with mount that I opened